beluga/test/beluga/algorithm/test_estimation.cpp

```diff
@@ -109,8 +106,7 @@ TEST_F(PoseCovarianceEstimation, PureTranslation) {
   const auto states = std::vector{SE2d{SO2d{0.0}, Vector2d{1.0, 2.0}}, SE2d{SO2d{0.0}, Vector2d{0.0, 0.0}}};
   const auto weights = std::vector(states.size(), 1.0);
   const auto [pose, covariance] = beluga::estimate(states, weights);
-  ASSERT_THAT(pose.translation(), Vector2Eq(Vector2d{0.5, 1.0}));
-  ASSERT_THAT(pose.so2(), SO2Eq(SO2d{0.0}));
+  ASSERT_THAT(pose, SE2Near(SO2d{0.0}, Vector2d{0.5, 1.0}, 0.001));
```

Comment:
@nahueespinosa nit^2: it would be nice to use named constants for tolerance, here and elsewhere. Mainly for readability's sake.

Reply:
Done!